default.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "deadlock-api-ingest";
+  version = "0.1.140-8659e00";
```

Comment:
is there a way to get this automatically? Or maybe we should add a sed command in the "Create Release" CI Job that edits this?